src/main/java/Duke.java

```diff
@@ -1,10 +1,267 @@
+import java.io.FileNotFoundException;
+import java.util.Scanner;
+import java.util.ArrayList;
+import java.io.File;
+import java.io.IOException;
+import java.io.FileWriter;
+import java.util.function.ToDoubleFunction;
+
 public class Duke {
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+
+        File dir;
+        File f;
+        ArrayList<Task> list = new ArrayList<>();
+
+        dir = new File("./data");
+        if (!dir.exists()) {
+            dir.mkdirs();
+        }
+        f = new File(dir, "duke.txt");
+        if (!f.exists()) {
+            try {
+                f.createNewFile();
+            } catch (IOException ex) {
+                System.out.println(ex.toString());
+            }
+        }
+        try {
+            list = readFileContents(f);
+        } catch (FileNotFoundException fe) {
+            System.out.println("File corrupted! Failed to create new file.");
+        }
+
+        String line = "_____________________________________________";
+        String greeting = "Hello! I'm Donald.\nWhat can I do for you?\n";
+        System.out.println(line);
+        System.out.println(greeting + line + "\n");
+
+        Scanner sc = new Scanner(System.in);
+        String input;
+        int index;
+        Task t;
```

Comment:
```suggestion
        Task task;
```
Perhaps the name can be spelt out fully, same as the above.